ci-operator/step-registry/cilium/conf/cilium-conf-commands.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+cilium_olm_rev=bdf0e8fd6c5e82708fe9c95adb0d3142e21fabe1
+cilium_version=1.9.5
+
+cat > "${SHARED_DIR}/manifest_cluster-network-03-config.yml" << EOF
```

Comment:
Not sure if its necessary - `cluster-network-02-config.yml` already has similar contents for config.openshift.io/v1 Network.

Reply:
I was staying on the explicit side of things... OVN workflow seem to do this, so I figured I'll do a similar thing here.